### PR TITLE
[#267,irods/irods#7495] Update for MySQL 8.4

### PR DESCRIPTION
**Pull request by SwooshyCueb**
Addresses #267
Addresses irods/irods#7495

### CODE REVIEW

**Review by alanking — APPROVED** [review]
Seems good to me. Let's get an approval from @korydraughn and/or @trel before #s.

File: docs/getting_started/installation.md
Comment (trel) [inline docs/getting_started/installation.md:82]: versions -> "unreleased versions" ?

Comment (korydraughn) [reply docs/getting_started/installation.md:82]: Where does it say they're unreleased?
Is it because they aren't considered GA (General Availability)?

**Review by korydraughn — APPROVED** [review]
Looks good to me.

Let's wait for @trel's response/approval.

File: docs/getting_started/installation.md
Comment (SwooshyCueb) [reply docs/getting_started/installation.md:82]: Yeah they're released. They're just not considered stable or GA.

Comment (trel) [reply docs/getting_started/installation.md:82]: Ah, then carry on.  Resolving.

**Comment by trel** [comment]
I'm go.

**Review by trel — APPROVED** [review]

**Comment by SwooshyCueb** [comment]
\#'d

**Pull request merged**
